cms/envs/common.py

```diff
@@ -2725,3 +2725,7 @@
 
 DISCUSSIONS_INCONTEXT_FEEDBACK_URL = ''
 DISCUSSIONS_INCONTEXT_LEARNMORE_URL = ''
+
+#### django-simple-history##
+# disable indexing on date field via django-simple-history.
+SIMPLE_HISTORY_DATE_INDEX = False
```

Comment:
https://github.com/cloudtruth/django-simple-history/blob/b45c6cc8442038b03be6ef1b3619120cecdd61f2/docs/historical_model.rst#indexed-history_date

Many queries use history_date as a filter. The as_of queries combine this with the original model's primary key to extract point-in-time snapshots of history. By default the history_date field is indexed. You can control this behavior using settings.py.

`#disable indexing on history_date`
```
SIMPLE_HISTORY_DATE_INDEX = False
```

Reply:
Why do we want to disable it?